omniport/core/base_auth/managers/get_user.py

```diff
@@ -56,20 +56,20 @@ def get_user(username):
         q_institute_webmail_address = Q(institute_webmail_address=username)
         q = (
                 q_primary_phone_number
-                | q_secondary_phone_number
```

Comment:
Why do you not want to query the secondary phone number?

Reply:
@pradumangoyal what was the reason?

Reply:
@dhruvkb secondary phone numbers are generally of the person parents at least for the student which is our major audience. So, in the case there are siblings studying this can collide. I am not sure if we should constraint it to be unique in the table.

Reply:
Even if we store parents number in their person's `primary_phone_number`, there is a chance that it collides with the students `secondary_phone_number`. Currently, We don't have a wat to check unique in `primary` and `secondary` collectively.

Reply:
Secondary phone numbers are "technically" supposed to be for people with dual SIMs. Parents' phone numbers belong in `Person` of parent relation -> `ContactInformation`.

Reply:
Agreed I was thinking according to the data we receive.  I was not sure what to give preference one more way to login over the way to register a new student. But finally, the ope source aspects should be given a preference.